test_conformance/printf/test_printf.cpp

```diff
@@ -825,73 +825,75 @@ int test_address_space_4(cl_device_id deviceID, cl_context context, cl_command_q
-    ADD_TEST( int_6 ),
-    ADD_TEST( int_7 ),
-    ADD_TEST( int_8 ),
-
-    ADD_TEST( float_0 ),
-    ADD_TEST( float_1 ),
-    ADD_TEST( float_2 ),
-    ADD_TEST( float_3 ),
-    ADD_TEST( float_4 ),
-    ADD_TEST( float_5 ),
-    ADD_TEST( float_6 ),
-    ADD_TEST( float_7 ),
-    ADD_TEST( float_8 ),
-    ADD_TEST( float_9 ),
-    ADD_TEST( float_10 ),
-    ADD_TEST( float_11 ),
-    ADD_TEST( float_12 ),
-    ADD_TEST( float_13 ),
-    ADD_TEST( float_14 ),
-    ADD_TEST( float_15 ),
-    ADD_TEST( float_16 ),
-    ADD_TEST( float_17 ),
-
-    ADD_TEST( float_limits_0 ),
-    ADD_TEST( float_limits_1 ),
-    ADD_TEST( float_limits_2 ),
-
-    ADD_TEST( octal_0 ),
-    ADD_TEST( octal_1 ),
-    ADD_TEST( octal_2 ),
-    ADD_TEST( octal_3 ),
-
-    ADD_TEST( unsigned_0 ),
-    ADD_TEST( unsigned_1 ),
-
-    ADD_TEST( hexadecimal_0 ),
-    ADD_TEST( hexadecimal_1 ),
-    ADD_TEST( hexadecimal_2 ),
-    ADD_TEST( hexadecimal_3 ),
-    ADD_TEST( hexadecimal_4 ),
-
-    ADD_TEST( char_0 ),
-    ADD_TEST( char_1 ),
-    ADD_TEST( char_2 ),
-
-    ADD_TEST( string_0 ),
-    ADD_TEST( string_1 ),
-    ADD_TEST( string_2 ),
-
-    ADD_TEST( vector_0 ),
-    ADD_TEST( vector_1 ),
-    ADD_TEST( vector_2 ),
-    ADD_TEST( vector_3 ),
-    ADD_TEST( vector_4 ),
-
-    ADD_TEST( address_space_0 ),
-    ADD_TEST( address_space_1 ),
-    ADD_TEST( address_space_2 ),
-    ADD_TEST( address_space_3 ),
-    ADD_TEST( address_space_4 ),
+    ADD_TEST(int_0),           ADD_TEST(int_1),
+    ADD_TEST(int_2),           ADD_TEST(int_3),
+    ADD_TEST(int_4),           ADD_TEST(int_5),
+    ADD_TEST(int_6),           ADD_TEST(int_7),
+    ADD_TEST(int_8),
+
+    ADD_TEST(float_0),         ADD_TEST(float_1),
+    ADD_TEST(float_2),         ADD_TEST(float_3),
+    ADD_TEST(float_4),         ADD_TEST(float_5),
+    ADD_TEST(float_6),         ADD_TEST(float_7),
+    ADD_TEST(float_8),         ADD_TEST(float_9),
+    ADD_TEST(float_10),        ADD_TEST(float_11),
+    ADD_TEST(float_12),        ADD_TEST(float_13),
+    ADD_TEST(float_14),        ADD_TEST(float_15),
+    ADD_TEST(float_16),        ADD_TEST(float_17),
+
+    ADD_TEST(float_limits_0),  ADD_TEST(float_limits_1),
+    ADD_TEST(float_limits_2),
+
+    ADD_TEST(octal_0),         ADD_TEST(octal_1),
+    ADD_TEST(octal_2),         ADD_TEST(octal_3),
+
+    ADD_TEST(unsigned_0),      ADD_TEST(unsigned_1),
+
+    ADD_TEST(hexadecimal_0),   ADD_TEST(hexadecimal_1),
+    ADD_TEST(hexadecimal_2),   ADD_TEST(hexadecimal_3),
+    ADD_TEST(hexadecimal_4),
+
+    ADD_TEST(char_0),          ADD_TEST(char_1),
+    ADD_TEST(char_2),
+
+    ADD_TEST(string_0),        ADD_TEST(string_1),
+    ADD_TEST(string_2),
+
+    ADD_TEST(vector_0),        ADD_TEST(vector_1),
+    ADD_TEST(vector_2),        ADD_TEST(vector_3),
+    ADD_TEST(vector_4),
+
+    ADD_TEST(address_space_0), ADD_TEST(address_space_1),
+    ADD_TEST(address_space_2), ADD_TEST(address_space_3),
+    ADD_TEST(address_space_4),
+
+    ADD_TEST(buffer_size),
```

Comment:
Should this be `ADD_TEST_VERSION(buffer_size, Version(1, 2))`, I think `CL_DEVICE_PRINTF_BUFFER_SIZE` was added in OpenCL 1.2

Reply:
Currently `InitCL` takes care of that for all the tests in the printf suite.

I agree though it'd be a nice change to make in general but also likely applies to a greater number of tests than just this one.

Reply:
ah missed that, thanks